### PR TITLE
Feature/design

### PR DESCRIPTION
**Pull request by Matthanson343**
## Types of changes

- [ ] Bug fix (non-breaking change which fixes an issue)
- [x] New feature (non-breaking change which adds documentation)
- [ ] Breaking change (This is a significant change)

## Checklist

- [ ] I have ensured that I have used relative links where appropriate.
- [ ] I have proofread my documentation.

### CODE REVIEW

File: design/design-tools/adobe-illustrator.md [hunk 1]
```diff
@@ -0,0 +1,16 @@
+# Adobe Illustrator
+
+## Create Icons
+
+Create vector illustrations and icons which can be easily exported to Adobe XD.
+
+
+## Save for Web
+
+Optimise vector illustrations and icons to reduce the file size and improve page load speeds.
```
Comment (dextermb) [inline design/design-tools/adobe-illustrator.md]: What optimizations do you normally do?

File: design/design-tools/adobe-illustrator.md [hunk 2]
```diff
@@ -0,0 +1,16 @@
...
+
+
+## Social Media Posts
+
+Use Illustrator as an alternative to Adobe Spark for more complex and unique designs.
+- [Social media post resolutions](design-tools/social-media.md)
```
Comment (dextermb) [inline design/design-tools/adobe-illustrator.md]: I think this link is broken?

```suggestion
- [Social media post resolutions](social-media.md)
```

**Comment by NilesB** [comment]
When talking about colours it may be worth mentioning how to choose colour pallets. For example it may be good to mention [Web Safe Colours](https://en.wikipedia.org/wiki/Web_colors) and limiting the number of shades of colours.

File: design/design-tools/adobe-illustrator.md (hunk 2 above)
Comment (Matthanson343) [reply design/design-tools/adobe-illustrator.md]: Fixed :)

Comment (dextermb) [reply design/design-tools/adobe-illustrator.md]: you should be able to click "commit suggestion" and it'll automatically commit it for you 👍

**Pull request merged**